### PR TITLE
Added Scala function to retrieve tile values and display in React modal

### PR DESCRIPTION
**Pull request by DarkManiel**
Initial PR for value_grid. Uncomment line 255 of CatalogService if no tile values are available for testing. Need to fix  close upon clicking 'x' in modal (works with 'close' but not 'x' for some reason).

### CODE REVIEW

File: server/src/main/scala/geotrellis/admin/server/CatalogService.scala [hunk 1]
```diff
@@ -55,14 +60,12 @@ object CatalogService extends ArgApp[CatalogArgs] with SimpleRoutingApp with Cor
...
                  val layerId = LayerId(layer, zoom)
-
                  timeOption match {
                    case Some(timeStr) =>
                      val time = DateTime.parse(timeStr)
-
                      catalog.loadTile(layerId, SpaceTimeKey(x, y, time))
                    case None =>
-
+
```
Comment (lossyrob) [inline server/src/main/scala/geotrellis/admin/server/CatalogService.scala]: Watch for adding whitespace.

**Comment by lossyrob** [comment]
Trying to run this, get a missing file error for Layers.jsx
Did you forget to add a file?

File: viewer/src/scripts/components/ValueGrid.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    		// 	}else{
+      //               console.log(vals[i]);
+    		// 		temp.push(vals[i]);    				
+    		// 	}
+    		// //}
+    		// mat.push(temp);
+    	}
+    	}
+    // 	return mat;
+    // }
```
Comment (lossyrob) [inline viewer/src/scripts/components/ValueGrid.jsx]: What is going on here?

**Comment by lossyrob** [comment]
I'm going to make a PR to your branch. It'll get it working with GeoTrellis master and gt-admin master. You'll have to `publish-local` on GeoTrellis, and then reingest rasters (I ran into some memory issues, so if you do, you should still probably get to go through enough zoom levels to do your work). Then you should clean up the code.

The way it currently stands, I can't move the map around. I can't merge this feature if it pops up a modal window every time I try to move the map. A much better solution would be to put the value viewer in the right pane, out of the way of the map. Please try to place it there.

**Comment by DarkManiel** [comment]
Okay thanks, just saw this. I'll make the changes you mentioned. I'll also look into why layers.jsx doesn't show up. I possibly missed it while adding.

File: viewer/src/scripts/components/ValueGrid.jsx (hunk 2 above)
Comment (DarkManiel) [inline viewer/src/scripts/components/ValueGrid.jsx]: Looks like my updated version didn't get pushed somehow. This is an old version. I'll push the updated files in a sec.

**Pull request closed without merging**